website/docusaurus.config.js

```diff
@@ -26,8 +26,8 @@ const linksSocial = [
 const config = {
   title: 'OpenLineage',
   tagline: 'OpenLineage',
-  url: 'https://openlineage.github.io',
-  baseUrl: '/',
+  url: 'https://pawel-big-lebowski.github.io', // TODO: replace with openlineage.io in the next step
+  baseUrl: '/openlineage-site', // TODO: replace with openlineage.io in the next PR
```

Comment:
I don't know much about configuration here but main page https://pawel-big-lebowski.github.io/openlineage-site/ does not work. Can this be adjusted so we can see all works correctly?

Reply:
According to https://docusaurus.io/docs/api/docusaurus-config#baseUrl `Always has both leading and trailing slash.`: probably it should be `/openlineage-site/`

Reply:
or: `baseUrl: process.env.NODE_ENV === 'production' ? '/openlineage-site/' : '/',` and run `NODE_ENV=production yarn build` in `openlineage-site` repo.

Reply:
I think the main page issue can be fixed with this -> https://github.com/OpenLineage/OpenLineage/pull/2961/files#diff-ca969f89131bf8aecb9ccf3a81ba10fea069104412c9733943fcbb8a0ee8275aR43
which will work both when server from domain without baseUrl and domain with `openlineage-site` as baseUrl. 

Production env is not necessary I believe. Once we switch prod into `openlineage.io`, `baseUril` will be removed and this should work the same way on localhost and on prod.

Reply:
This changed works for me locally and in your forked GH Pages.